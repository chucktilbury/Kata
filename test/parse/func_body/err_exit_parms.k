;-------------------------------------------------------------------------------
; Exit requires an expression
;-------------------------------------------------------------------------------

func name1()() {

	exit(list)
}
