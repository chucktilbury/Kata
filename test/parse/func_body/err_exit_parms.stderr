Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_exit_parms.k: 7: 11: expected an expression but got list
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_exit_parms.k: 9: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_exit_parms.k: 9: 1: expected function body but got end of file
